setup.py

```diff
@@ -100,7 +104,8 @@ def readme():
         'tests': tests_require,
         'flask': flask_require,
         'swagger-ui': swagger_ui_require,
-        'docs': docs_require
+        'docs': docs_require,
+        'uvicorn': uvicorn_requires,
```

Comment:
Install with `connexion[uvicorn]` to enable `app.run`. The `standard` extra includes the dependencies needed to watch files for reloading the app.